addons/website_mass_mailing/views/snippets_templates.xml

```diff
@@ -148,4 +135,22 @@
         </td>
     </xpath>
 </template>
+
+<template id="s_newsletter_popup_template" name="Newsletter block">
+    <section class="s_mm_popup_content container">
+        <div class="row">
+            <div class="col-12 col-md-6 pt112 pb112 o_mm_img_block oe_img_bg oe_custom_bg" style='background-image: url("/web/image/website.library_image_17");'>
+                <p>&amp;nbsp;</p>
+            </div>
+            <div class="col-12 col-md-6 pt112 pb112 o_mm_content_block">
```

Comment:
The mm_content_block class can be removed

Reply:
https://github.com/odoo-dev/odoo/commit/af9de05aca162bfc8106c61e001ba3273ef84a4e#diff-34ce1bd1849e1ffa6c277d5b87db7822R48 Here I pass this class as a selector so that the user can set background for this block.

Reply:
ok